build.cake

```diff
@@ -69,31 +69,39 @@ Task("deps").Does(() => {
     for (var i = 0; i < missingDepsCopy.Length; i++)
     {
         var dep = missingDepsCopy[i];
-        var depPathInOpenRA = depsInOpenRA[i];
+        var depPathInOpenRA = depsInOpenRA.SingleOrDefault(d => d.EndsWith(dep));
+        if (depPathInOpenRA == null)
+            throw new Exception(nameof(depsInOpenRA) + " does not contain an entry for " + dep);
```

Comment:
@abcdefg30 Please let me know if this fails to run for you.

I can remove the `nameof` easily I was just being lazy.